lib/widgets/dialogs/badge_dialog.dart

```diff
@@ -69,13 +70,13 @@ class _BadgeDialogState extends State<BadgeDialog> with SingleTickerProviderStat
 								textAlign: TextAlign.center
 							),
 							SizedBox(height: 6.0),
-							if(widget.badge.date != null)
-								Text(
-									AppLocales.of(context).translate('$_pageKey.earnedBadgeDate') + ': '
-										+ DateFormat.yMd(Localizations.localeOf(context).toString()).format(widget.badge.date),
-									style: Theme.of(context).textTheme.caption,
-									textAlign: TextAlign.center
-								),
+							// if(widget.badge.date != null)
+							// 	Text(
+							// 		AppLocales.of(context).translate('$_pageKey.earnedBadgeDate') + ': '
+							// 			+ DateFormat.yMd(Localizations.localeOf(context).toString()).format(widget.badge.date),
+							// 		style: Theme.of(context).textTheme.caption,
+							// 		textAlign: TextAlign.center
+							// 	),
```

Comment:
This will be used with different UIBadge (probably extended) on Child side. It's a date of badge assignment (this dialog will be used on caregiver and child side). I just prepared it in UI for later use.